pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -197,7 +203,9 @@ let
       "typing_extensions_archive"
       "wrapt"
       "zlib"
-    ];
+    ] ++ lib.optionals (!stdenv.isDarwin) [
+      "nsync" # fails to build on darwin
```

Comment:
Does this mean that the package fails to build on darwin entirely, or that it fails to build without this particular line of code? Or does it mean that the vendored version of nsync fails to build?

Reply:
I think it means that nsync in nixpkgs is not supported on darwin (darwin is not listed in platforms: https://github.com/NixOS/nixpkgs/blob/603db2cc147884acd25d0eb02cbf536e68c56bfb/pkgs/development/libraries/nsync/default.nix#L25-L27))

(This was taken from the tensorflow derivation.)